### DISCUSSION TITLE
2.0.0 release版本 官方没有区分BE和FB吗

### DISCUSSION THREAD

**Discussion by a137872798** [apache/doris#23596] [A - General / Q&A]
我看到2.0.0alpha版本是有区分BE和FE的，然后希望通过docker-compose进行部署。官方文档给的yml文件中分别需要be和fe的镜像。但是dockerhub上只能找到一个镜像。官网的下载页面也只有一个二进制包

**Comment by ixzc** [comment]
So far, the docker - compose only installed Doris: 2.0.0 _alpha version, other image has not been completed.

**Reply by a137872798** [reply]
thank you
